app/assets/javascripts/discourse/app/raw-views/topic-status.js

```diff
@@ -13,6 +13,7 @@ export default EmberObject.extend({
   @discourseComputed
   statuses() {
     const topic = this.topic;
+    const showPrivateMessageIcon = this.showPrivateMessageIcon;
```

Comment:
this const seems unneeded, it is only accessed once in the if statement. except for this extremely minor and optional nit, feel free to merge.